src/app/crypto.service.ts

```diff
@@ -172,4 +173,38 @@ export class CryptoService {
 
     return bs58check.encode(prefixAndKey);
   }
+
+  // NOTE: Our ETH address uses the bitcion/bitclout derivation path, not the ETH path.
+  // This is ugly but we only do it because they're just deposit addresses and we couldn't
+  // backfill this data for existing users because we store the derived private key.
+  // A user can still easily import their account to an ETH client to recover any ETH
+  // by coping the seedHex from local storage.
```

Comment:
Have we checked that this works?  Like that the seedhex in identity perfectly maps to the eth address that this function produces?

Reply:
Yep, verified

Reply:
Okay dope.  I feel pretty good about the addresses then :)